### PR TITLE
Update glm 0.9.9.1 and 0.9.9.2

### PR DESCRIPTION
**Pull request by caseymcc**
* I've followed [this guide](https://docs.hunter.sh/en/latest/creating-new/update.html)
  step by step carefully. Yes

* I've tested this package remotely and have excluded all broken builds.
  Here is the links to the Travis/AppVeyor with status "All passed":

  * https://travis-ci.org/caseymcc/hunter/builds/445645386>
  * https://ci.appveyor.com/project/caseymcc/hunter/builds/19755497

* This update will break few old toolchains.
  It doesn't break any toolchains however the update is ignored for Android as there is a compile error in the latest versions (https://github.com/g-truc/glm/issues/824#issuecomment-432645142) will be fixed in the next version.

### CODE REVIEW

File: cmake/configs/default.cmake [hunk 1]
```diff
@@ -232,7 +232,12 @@ hunter_default_version(glbinding VERSION 2.1.3-p0)
 hunter_default_version(glew VERSION 2.0.0-p1)
 hunter_default_version(glfw VERSION 3.3.0-p4)
 hunter_default_version(glib VERSION 2.54.0)
-hunter_default_version(glm VERSION 0.9.9.0)
+if(ANDROID)
+    #bug in the latest glm on Android
```
Comment (ruslo) [inline cmake/configs/default.cmake]: Please use indentation 2 as in the other parts of this file, e.g.
* https://github.com/ruslo/hunter/blob/v0.23.35/cmake/configs/default.cmake#L66-L72

Comment (caseymcc) [reply cmake/configs/default.cmake]: its changed

**Comment by ruslo** [comment]
Uploading:
* https://ci.appveyor.com/project/ingenue/hunter/builds/19762617
* https://travis-ci.org/ingenue/hunter/builds/445760818?utm_source=github_status&utm_medium=notification

**Comment by ruslo** [comment]
Released: https://github.com/ruslo/hunter/releases/tag/v0.23.36

**Pull request merged**